app/screens/settings/notification_settings/notification_settings.js

```diff
@@ -69,30 +58,25 @@ class NotificationSettings extends PureComponent {
     });
 
     goToNotificationSettingsEmail = () => {
-        if (Platform.OS === 'ios') {
-            const {currentUser, intl, navigator, theme} = this.props;
-            navigator.push({
-                backButtonTitle: '',
-                screen: 'NotificationSettingsEmail',
-                title: intl.formatMessage({
-                    id: 'mobile.notification_settings.email_title',
-                    defaultMessage: 'Email Notifications',
-                }),
-                animated: true,
-                navigatorStyle: {
-                    navBarTextColor: theme.sidebarHeaderTextColor,
-                    navBarBackgroundColor: theme.sidebarHeaderBg,
-                    navBarButtonColor: theme.sidebarHeaderTextColor,
-                    screenBackgroundColor: theme.centerChannelBg,
-                },
-                passProps: {
-                    currentUser,
-                    onBack: this.saveNotificationProps,
-                },
-            });
-        } else {
-            this.setState({showEmailNotificationsModal: true});
-        }
+        const {currentUser, intl, navigator, theme} = this.props;
+        navigator.push({
```

Comment:
I haven't test this, but this looks like is pushing a new screen for Android instead of just showing the modal as it was before, I'm I wrong?

Reply:
ok I see, cool!

anyway this PR needs a rebase

Reply:
Updated and done rebase.